### PR TITLE
Completed es6-destructuring exercise.

### PR DESCRIPTION
**Pull request by Edward-J-Kim**
Screenshot 2023-06-19 at 2 26 57 PM

### CODE REVIEW

File: es6-destructuring/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      const book2 = {
+        title: 'Les Fleurs du mal',
+        author: 'Charles Baudelaire',
+        libraryID: 2345,
+      };
+      const {
+        otherTitle = book2.title,
+        otherAuthor = book2.author,
+        otherLibraryID = book2.libraryID,
+      } = book2;
```
Comment (shawnkost) [inline es6-destructuring/index.html:38]: You shouldn't be doing `book2.title`, `book2.author` etc. You should just be doing the normal property names on the left side of the colon, and the variable names on the right side of the colon.

`const { title: title2 } = book2`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Edward!

File: es6-destructuring/index.html (hunk 1 above)
Comment (Edward-J-Kim) [reply es6-destructuring/index.html]: Screenshot 2023-06-19 at 2 43 59 PM
updated

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**